extensions/wikia/AdEngine/js/AdSlotBottomLeaderboard.run.js

```diff
@@ -11,9 +11,9 @@ require([
 		return false;
 	}
 
-	eventDispatcher.bind('ext.wikia.adEngine fillInSlot', function(slot, provider) {
+	eventDispatcher.bind('ext.wikia.adEngine fillInSlot', function(slotName, provider) {
 
-		if (slot[0].indexOf('LEADERBOARD') !== -1 && (slot[2] === 'Liftium' || provider.name === 'Later')) {
+		if (slotName.indexOf('LEADERBOARD') !== -1 && (provider.name === 'Liftium' || provider.name === 'Later')) {
```

Comment:
I guess You're right - this test might not be the best indicator as I don't really know the real complexity of a provider object, but it seems the differences are too small to matter. http://jsperf.com/lookingupobjects